ApplicationService/connection_pool.py

```diff
@@ -3,50 +3,56 @@
 from threading import Condition
 from typing import Callable
 from ApplicationService.repositories.identity import identity
-from ApplicationService.repositories.connectionInterface import cursor, connection, connection_pool
+from ApplicationService.repositories.connectionInterface \
```

Comment:
maybe you can avoid \ if use 

from package import (
 cursor,
 connection,
etc

)